CMakeLists.txt

```diff
@@ -341,11 +367,11 @@ if (WIN32 OR APPLE)
 else()
     include(cmake/external_tbb.cmake)
 endif()
-if (NGRAPH_DISTRIBUTED_ENABLE)
+
+if (NGRAPH_DISTRIBUTED_ENABLE AND NGRAPH_DISTRIBUTED_MLSL_ENABLE)
```

Comment:
Do you need to check NGRAPH_DISTRIBUTED_ENABLE? Looks like you set it to true if NGRAPH_DISTRIBUTED_MLSL_ENABLE is set.